File: python/a04.py
# A04 Binary Representation 1
# submission: https://atcoder.jp/contests/tessoku-book/submissions/38326161

n = int(input())
ans = []

for i in range(10)[::-1]:
    ans.append(str(int(bool(n & (1 << i)))))
print(''.join(ans))

File: rust/tessoku-book/src/bin/a04.rs
// A04 Binary Representation 
// submission: https://atcoder.jp/contests/tessoku-book/submissions/38326394

use proconio::input;

fn main() {
    input!{
        n: u32,
    }
    for i in (0..10).rev() {
        if n & (1 << i) == 0 {
            print!("0");
        } else {
            print!("1");
        }
    }
    print!("\n");
}
